modules/cluster-logging-deploy-clo-cli.adoc

```diff
@@ -0,0 +1,264 @@
+  collection:
+    logs:
+      type: "fluentd"  <8>
+      fluentd: {}
+----
+<1> The name must be `instance`.
+<2> The Cluster Logging management state. In most cases, if you change the default Cluster Logging defaults, you must set this to `Unmanaged`.
+However, an unmanaged deployment does not receive updates until Cluster Logging is placed back into the `Managed` state. For more information, see *Changing cluster logging management state*.
+<3> Settings for configuring Elasticsearch. Using the CR, you can configure shard replication policy and persistent storage. For more information, see *Configuring Elasticsearch*.
+<4> Specify the number of Elasticsearch nodes. See the note that follows this list.
+<5> Specify that each Elasticsearch node in the cluster is bound to a Persistent Volume Claim.
+<6> Settings for configuring Kibana. Using the CR, you can scale Kibana for redundancy and configure the CPU and memory for your Kibana nodes. For more information, see *Configuring Kibana*.
+<7> Settings for configuring Curator. Using the CR, you can set the Curator schedule. For more information, see *Configuring Curator*.
+<8> Settings for configuring Fluentd. Using the CR, you can configure Fluentd CPU and memory limits. For more information, see *Configuring Fluentd*.
+endif::[]
++
+[NOTE]
++
+====
+The maximum number of Elasticsearch master nodes is three. If you specify a `nodeCount` greater than `3`, {product-title} creates three Elasticsearch nodes that are Master-eligible nodes, with the master, client, and data roles. The additional Elasticsearch nodes are created as Data-only nodes, using client and data roles. Master nodes perform cluster-wide actions such as creating or deleting an index, shard allocation, and tracking nodes. Data nodes hold the shards and perform data-related operations such as CRUD, search, and aggregations. Data-related operations are I/O-, memory-, and CPU-intensive. It is important to monitor these resources and to add more Data nodes if the current nodes are overloaded.
+
+For example, if `nodeCount=4`, the following nodes are created:
+
+----
+$ oc get deployment
+
+cluster-logging-operator       1/1     1            1           18h
+elasticsearch-cd-x6kdekli-1    1/1     1            0           6m54s
+elasticsearch-cdm-x6kdekli-1   1/1     1            1           18h
+elasticsearch-cdm-x6kdekli-2   1/1     1            0           6m49s
+elasticsearch-cdm-x6kdekli-3   1/1     1            0           6m44s
+----
+
+The number of primary shards for the index templates is equal to the number of Elasticsearch data nodes.
+====
+
+.. Run the following command to create the instance:
++
+----
+$ oc create -f <file-name>.yaml
+----
++
+For example:
++
+----
+$ oc create -f clo-instance.yaml
+----
+
+. Verify the install by listing the pods in the *openshift-logging* project.
++
+You should see several pods for Cluster Logging, Elasticsearch, Fluentd, and Kibana similar to the following list:
```

Comment:
s/pods/Pods/
I know we are inconsistent on capitalizing this but: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-formatting

Reply:
outdated